### ISSUE TITLE
PDM Order From 'Custom' and 'Import' Category

### ISSUE THREAD

**Issue by shelby0341** [DSRP-Inc/DSRP_Tickets#349]
I'm not sure how the PDM system works but was told the boss menu was different? Johnny Mendez is reporting that he cannot promote people via the boss menu, he said he can only set as "employee" which is recruit level 0, or boss, but no pay grades in between.

**Comment by Daemon53** [comment]
Yeah, it’s part of that vehicle selling script. I told him I’d take a look at it. We may have to give him the other boss menu too.

**Comment by Daemon53** [comment]
- [x] Weaponize Veh - Issi Future Shock
- [x] Remove Imports and Custom categories
- [x] Setup new boss menu

**Comment by Daemon53** [comment]
Remove these people:
Mr Green, Angel Arizona, Hunter Boyce, Daemon Foster. those are the Character names of people that I can't fire because they arent ever around

**Comment by shelby0341** [comment]
Boss menu is set up
People removed from staff

need to check on the issi future shock being removed and the imports and customs categories

**Comment by shelby0341** [comment]
PDM can still order from the imports and customs categories (which btw the Imports include the "evo9" "r32" "rowerwodny" and "s15") which are branded cars that might still be in some files somewhere?

**Comment by Daemon53** [comment]
I need to delete from the DB. The cars are not actually in the server.
